### PR TITLE
Adds Serbian translation

### PR DESCRIPTION
**Pull request by marxo**
Adds a complete Serbian translation of the workshopper-adventure

### CODE REVIEW

File: examples/i18n/sr.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "title": "РАДИОНИЧАР ПРИМЕР РАДИОНИЦЕ",
+  "subtitle": "\u001b[23mИзаберите вежбу и притисните \u001b[3mEnter\u001b[23m да почнете",
+  "menu": {
+    "credits": "ЗАСЛУГЕ"
+  },
+  "exercise": {
+    "foo": "FOO",
+    "bar": "BAR",
+    "baz": "BAZ"
+  }
```
Comment (martinheidegger) [inline examples/i18n/sr.json:11]: foo, bar, baz?

**Comment by marxo** [comment]
There isn't an actual meaningful translation for those, and they are generally known amoungst developers in English form.

**Comment by martinheidegger** [comment]
Okay ^^

**Comment by martinheidegger** [comment]
Thank you for the translation effort, I merged the translation without asking you for a second review by someone else (even though we usually do) because the International Day is coming up soon. Please try to have another person review it ^^

**Comment by marxo** [comment]
Thanks! Sure thing, we'll have someone from the Belgrade chapter review it and we'll check again ourselves and possibly fix any mistakes that might have occurred.

**Comment by vuk-nikolic** [comment]
@martinheidegger just to let you know that @marxo translation is legit 👍 

no need to worry :)

**Pull request merged**